src/NumberInput.css

```diff
@@ -1,3 +1,30 @@
 .NumbInputField {
+  align-self: center;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+
   margin-top: 20px;
```

Comment:
Wouldn't it be better to use rem here?

Reply:
i agree with adriana

Reply:
The gap is kind of provisional. But true, I still should have used rem.